docs/oss-syntax.md

```diff
@@ -0,0 +1,348 @@
+##  OSS Syntax
+
+ 
+Rule format is inspired by CSS, but extended to support nesting and chaining and tweaked to deal with identifiers containing “.”s. 
+
+**CSS**
+```css
+  div#toc a {
+      text-decoration:underline;
+  }
+```
+
+
+**OSS**
+ ```css
+     class=User field=email {
+         cueTip:”Your email”;
```

Comment:
`cueTip` seems to be domain specific, `altText` or `placeholder` would be preferred